CHANGELOG.md

```diff
@@ -6,15 +6,26 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-### Added
+### **Added**
```

Comment:
I don't think there's value in changing formatting of headers. They're already level 3 headers, they don't need to be bold.